app/src/main/res/layout-land/profile_reset_pin_activity.xml

```diff
@@ -54,7 +54,6 @@
         android:scrollbars="none"
         android:paddingBottom="168dp"
         android:clipToPadding="false"
-        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
we need to remove all the constraints as `ScrollView` has no effect of it.

Reply:
ok

Reply:
Just for the future reference, the conversation should be resolve by PR reviewer just to confirm the required changes are done.

Reply:
ok I will take care about that . Thank you